doc/PreparingSpecifying/ConfigureProxy/ConfigureProxy.md

```diff
@@ -0,0 +1,39 @@
+# Configure the Proxy on TEF Laptop (Windows)

```

Comment:
The content looks good.. but i am just wondering where this file has been referred in PreparingSpecifying/PreparingSpecifying.md. 
If not, we need to include this file path in PreparingSpecifying.md

Reply:
added a link to the page